ports/espressif/supervisor/port.c

```diff
@@ -285,6 +284,7 @@ safe_mode_t port_init(void) {
     #endif
 
     #if ENABLE_JTAG
+    ESP_LOGI(TAG, "Marking JTAG pins never_reset")
```

Comment:
Waste of binary space?

Reply:
This is only included in `debug` builds.